test/helpers/kubectl.go

```diff
@@ -892,7 +892,7 @@ func (kub *Kubectl) GetPods(namespace string, filter string) *CmdRes {
 // returns an error if pods cannot be retrieved correctly
 func (kub *Kubectl) GetPodsNodes(namespace string, filter string) (map[string]string, error) {
 	jsonFilter := `{range .items[*]}{@.metadata.name}{"="}{@.spec.nodeName}{"\n"}{end}`
-	res := kub.Exec(fmt.Sprintf("%s -n %s get pods %s -o jsonpath='%s'",
+	res := kub.Exec(fmt.Sprintf("%s -n %s get pods -l %s -o jsonpath='%s'",
```

Comment:
would it make sense to quote it the same way as with jsonpath?

Reply:
I guess it wouldn't hurt, I'll do that :+1: